app/routes/socialEvents.tsx

```diff
@@ -0,0 +1,15 @@
+import EventCard from '~/components/common/eventCard';
+
+export default function socialEvents() {
+  return (
+    <>
+      <EventCard
```

Comment:
You can pass the whole event object to this component and destructure the required values inside so that later if the requirement changes and we need to remove or add any details we wont have to add or remove props.

Reply:
Done, thanks!